site/content/contextual-data-platform/tutorials/evaluate-locally.md

```
@@ -0,0 +1,625 @@
+  --from-literal=license-client-secret="<license-client-secret>"
+```
+
+Verify that the secret was created:
+
+```sh
+kubectl get secret arango-license-key --namespace arango
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)